tests/dotnetexample/README.md

```diff
@@ -1,10 +1,12 @@
 ## .Net example
```

Comment:
Just a note on usage here, it doesn't affect this PR: `.Net` should be `.NET` wherever we use it